include/zephyr/bluetooth/bluetooth.h

```diff
@@ -517,6 +525,60 @@ size_t bt_data_get_len(const struct bt_data data[], size_t data_count);
  */
 size_t bt_data_serialize(const struct bt_data *input, uint8_t *output);
 
+struct bt_le_local_features {
+	/**
+	 * @brief Local LE controller supported features.
+	 *
+	 * Refer to BT_LE_FEAT_BIT_* for values.
+	 * Refer to the BT_FEAT_LE_* macros for value comparionson.
+	 * See Bluetooth Core Specification 6.0, Vol 6, Part B, Section 4.6 table 4.10
+	 */
+	uint8_t features[BT_LE_LOCAL_SUPPOTRED_FEATURES_SIZE];
+
+	/** Local LE controller supported states
+	 *
+	 * Refer to BT_LE_STATES_* for values.
+	 * See Bluetooth Core Specification 6.0, Vol 4, Part E, Section 7.8.27
+	 */
+	uint64_t states;
+
+	/** ACL data packet length */
+	uint16_t acl_mtu;
+	/** Total number of ACL data packets */
+	uint8_t acl_pkts;
+
+	/** ISO data packet length */
+	uint16_t iso_mtu;
+	/** Total number of ISO data packets */
+	uint8_t iso_pkts;
+
+	/** Size of the controller resolving list */
```

Comment:
The actual size can be smaller depending on controller implementation
```suggestion
	/** Maximum size of the controller resolving list.
	  * 
	  * See Bluetooth Core Specification, Vol 6, Part E, Section 7.8.41.
	  */
```